CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 - Support the full unicode character set via UTF-8 encoding for metric tags submitted via the statsd format. Certain restricted characters require escape sequences, see [docs](https://develop.sentry.dev/sdk/metrics/#normalization) for the precise rules. ([#3358](https://github.com/getsentry/relay/pull/3358))
 - Stop extracting count_per_segment and count_per_op metrics. ([#3380](https://github.com/getsentry/relay/pull/3380))
 
+**Bug Fixes**:
+
+- Allow spans with exclusive_time 0. ([#3385](https://github.com/getsentry/relay/pull/3385))
```

Comment:
This doesn't just allow spans with exclusive time `0` it allows them with missing exclusive time.